gui/resources.xml

```diff
@@ -1234,19 +1272,11 @@ Vocabulary
       <!-- {{{ Layers panel -->
       <child>
         <object class="GtkToggleAction" id="ToggleLayersTool">
-          <property name="label" translatable="yes" context="Menu→Window (labels), Toolbar (labels), Accel Editor (labels)">Layers Panel</property>
-          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Toggle the Layers dockpanel (arrange layers, or assign modes).</property>
+          <property name="label" translatable="yes" context="Menu→Layers (lables), Menu→Window (labels), Toolbar (labels), Accel Editor (labels)">Layers Panel</property>
+          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Dockable panel for managing layers.</property>
           <property name="icon-name">mypaint-layers-symbolic</property>
           <signal name="activate" handler="toggle_dockpanel_cb"/>
         </object>
-      </child>
-      <child>
-        <object class="GtkAction" id="RevealLayersTool">
-          <property name="label" translatable="yes" context="Menu→Layers (labels), Accel Editor (labels)">Show Layers Panel</property>
-          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Reveal the Layers dockpanel (arrange layers, or assign modes).</property>
-          <property name="icon-name">mypaint-layers-symbolic</property>
-          <signal name="activate" handler="reveal_dockpanel_cb"/>
-        </object>
```

Comment:
Ahhh that makes much more sense. So I will need to clarify the tooltips for the reveal versions that they are mainly used in fullscreen mode than and don't reset window.

This is a really confusing way to handle the panels cause what works with one will not work with the other and vice a versa.  Probably need to add a FIXME tag so it can be addressed in the future. I will revert the reveal version to the default keys, but I will also need to assigned the same keys for the toggle, but instead have a modifier attached to them instead.

Thanks you for clarifying that @0ion9. ^^